.github/workflows/cli.yml

```diff
@@ -93,10 +94,10 @@ jobs:
 
       - uses: actions/download-artifact@65a9edc5881444af0b9093a5e628f2fe47ea3b2e # v4.1.7
         with:
-          name: sops-linux-amd64-${{ github.sha }}
+          name: sops-1.22-linux-amd64-${{ github.sha }}
```

Comment:
It's likely better to use a one-element matrix to move these versions out from the steps.

Reply:
There is no matrix in this job (test) 🤔
You see that the build job is just required for the test job, after a successful build both versions are available.

Reply:
I know, but you could add a matrix for `test` as well, to move the versions out from the steps into the matrix.

Reply:
I added 1.22 to a new test "matrix".